backbone.js

```diff
@@ -394,6 +394,7 @@
   var Model = Backbone.Model = function(attributes, options) {
     var attrs = attributes || {};
     options || (options = {});
+    _.extend(this, _.result(this, 'localProps'));
```

Comment:
We shouldn't need `result` for this, no? There's never a case `localProps` would be an object over just sticking them on the prototype, yeah?